spec/requests/create_dro_spec.rb

```diff
@@ -105,6 +106,7 @@
       expect(IngestJob).to have_received(:perform_later).with(model_params: expected_model_params,
                                                               background_job_result: instance_of(BackgroundJobResult),
                                                               signed_ids: { 'file2.txt' => signed_id },
+                                                              globus_ids: {},
```

Comment:
Seeing that you had to add this in a few different spots, I wonder if these jobs ought to default to `globus_ids: {}` in the `#perform` method definition to cut down on needing all this setup?